### PR TITLE
add wallaby

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by bcherny** [comment]
Nice! Does this mean we can get rid of Karma?

**Comment by ghost** [comment]
good point - test runners are different (i.e. karma uses chrome and wallaby electron)... and wallaby only works in editors, not CI atm

they're still working on it: https://github.com/wallabyjs/public/issues/72

**Comment by bcherny** [comment]
I see. Lgtm

**Review by bcherny — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "@types/angular-mocks": "^1.5.9",
     "@types/jasmine": "^2.5.46",
     "browserify": "^14.0.1",
+    "electron": "^1.6.2",
```
Comment (ghost) [inline package.json:40]: for future reference, using electron-prebuilt is probably better

**Pull request merged**